packages/graphqlsp/src/ast/checks.ts

```diff
@@ -93,10 +93,13 @@ export const isGraphQLTag = (
 /** Retrieves the `__name` branded tag from gql.tada `graphql()` or `graphql.persisted()` calls */
 export const getSchemaName = (
   node: ts.CallExpression,
-  typeChecker: ts.TypeChecker | undefined
+  typeChecker: ts.TypeChecker | undefined,
+  isTadaPersistedCall = false
 ): string | null => {
   if (!typeChecker) return null;
-  const type = typeChecker.getTypeAtLocation(node.expression);
+  const type = typeChecker.getTypeAtLocation(
+    isTadaPersistedCall ? node.getChildAt(0).getChildAt(0) : node.expression
```

Comment:
Might be a bit easier to read

```suggestion
    // When we are on a graphql.persisted call we need
    // to access the "graphql" part of the expression
    isTadaPersistedCall ? node.expression.expression : node.expression
```